reactor-core/src/main/java/reactor/core/publisher/doc-files/marbles/join.svg

```diff
@@ -1,25 +1,15 @@
-<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="-1 20 615 572" width="615" height="572" id="svg339">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="-1 20 610.71 582" width="610.71" height="582" id="svg339">
```

Comment:
file-level comments:
this is not what I had in mind. Putting `onNext` on these arrows is plain wrong, because the graphical convention is that a vertical bar is an `onComplete` signal.
what I had in mind was to edit ONE of these inner green flux to represent the "stop on onNext" case:
 - extend the horizontal line, moving the onComplete bar further right
 - add a small symbol (like a filled green square) to represent an onNext right at the spot where the onComplete bar used to be
 - connect the vertical dotted line to said square rather than the onComplete bar

Doing that only for one inner flux will demonstrate the behavior in case of `onNext` while keeping things visually simpler with `onComplete` case for all the others.

Also, can you confirm you've followed the instructions to "compress" the SVG? (see https://github.com/reactor/reactor-core/blob/master/docs/svg/README.md#optimizing-the-svg-files)

Reply:
You are right, I missed it. I improved the diagram so far, hope its better now. 

Yes, I used `svgo` to compress it ;)